APE13.rst

```diff
@@ -498,3 +478,7 @@ Decision rationale
 ------------------
 There were extensive discussions with multiple stakeholders undertaken in the process of putting together this APE and it appears as though concensus has been reached. This APE was accepted on Dec 18, 2017.
 
+Previous versions of this APE
+-----------------------------
+
```

Comment:
It might be good to briefly mention that the previous version included a vision for `specvis` and describe the rationale for removing it.

Reply:
Fair point, I'll try to get something added today.

Reply:
Actually, I'm going to give a bit of push-back first. This section is formatted identically to all other APEs that have a previous version. In other words, no other APE provides any discussion of the contents of the previous version. Presumably, the previous version is still readable in its entirety.

Reply:
I don't think any other revision did more than clarify. Though arguably that is the case here too. Anyway, it is no big deal, I think right now this is just ensuring the APE reflects reality, not a wish from the past, and that is fine! Happy to have this in as is too.

Reply:
Thanx. I'll leave this open in case others want to comment.